shared/wallets/routes.js

```diff
@@ -114,3 +118,79 @@ const routeTree = () => {
 }
 
 export default routeTree
+
+const walletsSubRoutes = isMobile
+  ? {}
+  : {
+      createNewAccount: {getScreen: () => require('./create-account/container').default},
+      exportSecretKey: {getScreen: () => require('./export-secret-key/container').default},
+      linkExisting: {getScreen: () => require('./link-existing/container').default},
+      reallyRemoveAccount: {getScreen: () => require('./wallet/settings/popups').ReallyRemoveAccountPopup},
+      receive: {getScreen: () => require('./receive-modal/container').default},
+      removeAccount: {getScreen: () => require('./wallet/settings/popups').RemoveAccountPopup},
+      renameAccount: {getScreen: () => require('./wallet/settings/popups').RenameAccountPopup},
+      setDefaultAccount: {getScreen: () => require('./wallet/settings/popups').SetDefaultAccountPopup},
+      setInflation: {getScreen: () => require('./wallet/settings/popups').InflationDestination},
+      settings: {getScreen: () => require('./wallet/settings/container').default},
+      transactionDetails: {getScreen: () => require('./transaction-details/container').default},
+      wallet: {getScreen: () => require('./wallet/container').default},
+    }
+
+class WalletsSubNav extends React.PureComponent<any> {
+  render() {
+    const navigation = this.props.navigation
+    const index = navigation.state.index
+    const activeKey = navigation.state.routes[index].key
+    const descriptor = this.props.descriptors[activeKey]
+    const childNav = descriptor.navigation
+    const WalletsAndDetails = require('./wallets-and-details/container').default
+
+    return (
+      <Kb.Box2 direction="horizontal" fullHeight={true} fullWidth={true}>
+        {/* $FlowIssue */}
+        <WalletsAndDetails>
+          <SceneView
```

Comment:
yeah

Reply:
@MarcoPolo answering or do you have the same question? @chrisnojima and I talked IRL yesterday I thought the answer was no.

Reply:
Oh, sorry I misunderstood your question. I thought you meant that you were wondering how to display another scene here and if it was like how it was handled in the app/modal stuff like [this](https://github.com/keybase/client/pull/15577/files/8b6a888fbd50e7c5069fadfda4e7ed73589d46ec#diff-48a273a931fbc8da057c75f10207c7d9R80). I was answering yes, if you wanted to display multiple scenes (from different navigators) you could do something similar.

What you actually asked was that modals don't appear when you are here. That sounds like a bug, since we shouldn't have to do anything here to make modals appear over the app. That should all be handled in the code I linked above. Sorry for the confusion!